src/app/control-planes/components/OnboardingAlert.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <KAlert
+    v-if="isShowingOnboardingAlert"
+    appearance="success"
+    dismiss-type="icon"
+    data-testid="onboarding-notification"
+    @closed="closeAlert"
+  >
+    <template #alertMessage>
+      <div class="onboarding-alert-content">
+        <!-- eslint-disable-next-line vue/no-v-html  -->
+        <div v-html="t('main-overview.detail.onboarding.message', { name: t('common.product.name') })" />
+
+        <KButton
+          appearance="primary"
+          size="small"
+          class="action-button"
+          :to="{ name: 'onboarding-welcome' }"
+        >
+          {{ t('main-overview.detail.onboarding.get_started_link') }}
+        </KButton>
+      </div>
+    </template>
+  </KAlert>
+</template>
+
+<script lang="ts" setup>
+import { ref } from 'vue'
+
+import { useI18n } from '@/utilities'
+import { ClientStorage } from '@/utilities/ClientStorage'
+
+const { t } = useI18n()
+
+const isShowingOnboardingAlert = ref(ClientStorage.get('hasDismissedOnboardingAlert') !== 'true')
+
+function closeAlert() {
+  isShowingOnboardingAlert.value = false
+  ClientStorage.set('hasDismissedOnboardingAlert', 'true')
+}
+</script>
+
+<style lang="scss" scoped>
+.onboarding-alert-content {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  gap: $kui-space-60;
+}
+</style>
```

Comment:
If we are maintaining this component as a separate component then this file should go in the onboarding module not the control-planes module because we said we would locate these sorts of file based on "What they are".

I'm just taking another look at this PR as a whole as I see there have been quite a few changes recently, I just wanted to leave this note here before I look at it again.

Reply:
I’ve moved the component into the onboarding module now.